collect-artifacts.js

```diff
@@ -1,18 +1,15 @@
-var fs = require('fs')
-var path = require('path')
 var error = require('./error')
+var recursive = require('recursive-readdir')
 
 module.exports = collectArtifacts
 
 function collectArtifacts (release, opts, cb) {
   var fileExp = opts['include-regex']
-  fs.readdir(release, function (err, files) {
+  recursive(release, function (err, files) {
```

Comment:
Does `recursive-readdir` yield relative or absolute paths? If relative, we need to keep the `.map()` call below. If absolute, the `fileExp` regular expression will no longer work as expected and we need to fix that.

Reply:
It's not absolute,  but it includes the directory already. In this case each filename in `file` in `files` has `release` already pretended. Testing with my setup , the map causes the `release` to be prepended twice.
As for the fileExp I think it should work as expected, especially with the default value which includes '$' .

Reply:
So before this, the regexp would be tested against e.g. a filename `foobar`. Now it will be tested against `release/foobar`. Am I right? I'm not worried about the default value, but about people that use a custom regex.

Reply:
Yes, you are right. 
I can't really think of a case where it might be problematic,  but people tend to get really creative with regexes :)
I  guess, I can add something so the regex works on the same portion of the filename.